src/modules/sub-items/sub.items.module.js

```diff
@@ -335,6 +360,188 @@ export default class SubItemsModule extends Component {
         });
     }
 
+    onMoveBtnClick() {
+        this.toggleUdw(true);
+    }
+
+    bulkMove(location) {
+        const { restInfo } = this.props;
+        const itemsToMove = this.getSelectedItems();
+        const contentsToMove = itemsToMove.map((item) => item.location);
+
+        bulkMoveLocations(restInfo, contentsToMove, location._href, this.afterBulkMove.bind(this, location));
+    }
+
+    afterBulkMove(location, movedLocations, notMoved) {
```

Comment:
I renamed it to `notMovedLocations` so that it will be similar to `movedLocations` and won't suggest that it is `item === { location: {...}, conent: {...} }`.